packages/rocketchat-file-upload/server/methods/sendFileMessage.js

```diff
@@ -53,11 +53,7 @@ Meteor.methods({
 			rid: roomId,
 			ts: new Date(),
 			msg: '',
-			file: {
-				_id: file._id,
-				name: file.name,
-				type: file.type
-			},
+			file,
```

Comment:
What's the reason for doing this? There are a lot more fields on the `file` object which we don't want on the message as that's yet another place to have to update the data if the file changes somehow..